apps/web/public/static/locales/en/common.json

```diff
@@ -121,6 +121,9 @@
   "team_upgrade_banner_description": "Thank you for trialing our new team plan. We noticed your team \"{{teamName}}\" needs to be upgraded.",
   "team_upgrade_banner_action": "Upgrade here",
   "team_upgraded_successfully": "Your team was upgraded successfully!",
+  "org_upgrade_banner_description": "Thank you for trialing our Organization plan. We noticed your Organization \"{{teamName}}\" needs to be upgraded.",
+  "org_upgrade_banner_action": "Upgrade here",
+  "org_upgraded_successfully": "Your Organization was upgraded successfully!",
```

Comment:
To avoid having all these new translation keys we could also add an entity variable here with either Team or Organization. Like in that PR: https://github.com/calcom/cal.com/pull/9184/files#r1213441386